src/core/mavsdk_impl.cpp

```diff
@@ -345,10 +351,15 @@ uint8_t MavsdkImpl::get_own_component_id() const
 uint8_t MavsdkImpl::get_mav_type() const
 {
     switch (_configuration.load()) {
+        case Mavsdk::Configuration::Autopilot:
+            return MAV_TYPE_GENERIC;
```

Comment:
I wonder what that means.

Reply:
"Generic micro air vehicle": https://mavlink.io/en/messages/common.html#MAV_TYPE_GENERIC

:sweat_smile: